em_examples/Base.py

```diff
@@ -15,7 +15,7 @@ def __init__(self, widgets, kwargs):
 
     @property
     def kwargs(self):
-        instanceCheck = lambda x: isinstance(x, ToggleButtons) or isinstance(x, FloatSlider) or isinstance(x, IntSlider) or isinstance(x, FloatText) or isinstance(x, fixed) or isinstance(x, Checkbox)
+        instanceCheck = lambda x: isinstance(x, ToggleButton) or isinstance(x, ToggleButtons) or isinstance(x, FloatSlider) or isinstance(x, IntSlider) or isinstance(x, FloatText) or isinstance(x, fixed) or isinstance(x, Checkbox)
```

Comment:
The problem I see with this is that some apps have "hidden inputs" defined through ipywidgets.fixed(). The above check would miss it. That would mean that we cannot use anymore the app.kwargs as an input for the function inside the app, which is its main use I feel.
![dc_overburden_2_5d 2017-05-31 15-54-52](https://cloud.githubusercontent.com/assets/13682747/26657813/135b39b8-461b-11e7-8856-a29b59e6e224.png)

![dc_overburden_2_5d 2017-05-31 16-02-50](https://cloud.githubusercontent.com/assets/13682747/26657800/fc511760-461a-11e7-90e2-897a23494ed0.png)

My conpromise function would then looks like: 
instanceCheck = lambda x: isinstance(x, ipywidgets.widget.Widget) or isinstance(x, ipywidgets.fixed)

which then allow:
![dc_overburden_2_5d 2017-05-31 16-05-12](https://cloud.githubusercontent.com/assets/13682747/26657839/3a3b6724-461b-11e7-943e-81d92123be19.png)